Modules/Loadable/Markups/Resources/Schema/markups-schema-v1.0.1.json

```diff
@@ -0,0 +1,532 @@
+                "description": "URL of versioned schema."
+            },
+            "markups": {
+                "$id": "#markups",
+                "type": "array",
+                "title": "Markups",
+                "description": "Stores position and display properties of one or more markups.",
+                "additionalItems": true,
+                "items": {
+                    "$id": "#markupItems",
+                    "anyOf": [
+                        {
+                            "$id": "#markup",
+                            "type": "object",
+                            "title": "Markup",
+                            "description": "Stores a single markup.",
+                            "default": {},
+                            "required": ["type"],
+                            "additionalProperties": true,
+                            "properties": {
+                                "type": {
+                                    "$id": "#markup/type",
+                                    "type": "string",
+                                    "title": "Basic type",
+                                    "enum": ["Fiducial", "Line", "Angle", "Curve", "ClosedCurve", "Plane"]
+                                },
+                                "name": {
+                                    "$id": "#markup/name",
+                                    "type": "string",
+                                    "title": "Name",
+                                    "description": "Displayed name of the markup.",
+                                    "default": ""
+                                },
+                                "coordinateSystem": {
+                                    "$id": "#markup/coordinateSystem",
+                                    "type": "string",
+                                    "title": "Control point positions coordinate system name",
+                                    "description": "Coordinate system name. Medical images most commonly use LPS patient coordinate system.",
+                                    "default": "LPS",
+                                    "enum": ["LPS", "RAS"]
+                                },
+                                "coordinateUnits": {
+                                    "$id": "#markup/coordinateUnits",
+                                    "anyOf": [
+                                        {
+                                            "type": "string",
+                                            "title": "Units of control point coordinates",
+                                            "description": "Control point coordinate values are specified in this length unit. Specified in UCUM.",
+                                            "default": "mm",
+                                            "enum": ["mm", "um"]
+                                        },
+                                        {
+                                            ttt
```

Comment:
Remove?

Reply:
Good catch thank you! It seems I've not committed the final version.